docs/src/config/ini-config.adoc

```diff
@@ -423,33 +424,43 @@ must write RS274NGC code to standard output. This output is what will
 be displayed in the text area, previewed in the display area, and
 executed by LinuxCNC when Run.
 
-* 'PROGRAM_EXTENSION = .extension Description'
+----
+PROGRAM_EXTENSION = .extension Description
+----
```

Comment:
I would prefer to leave it as it was before, because now it is inconsistent - some options are formatted as code and the more are formatted as item list

Reply:
This is not an option but actually the pattern of the statements in the [FILTER] section, both of the INI file and here in the doc.

Reply:
OK, changed this one to single list entry for consistency

Reply:
>This is not an option but actually the pattern of the statements in the [FILTER] section, both of the INI file and here in the doc.

You can also call it setting or value...
> Also, I suspect it to be wrong, missign a '#' in front of 'Description'.

No that is the syntax. `<EXTENSION><SPACE><DESCRIPTION>`
And furthermore it would be an incorrect use of commenting, see https://linuxcnc.org/docs/2.9/html/config/ini-config.html#_comments